packages/packages/commonwell-sdk/src/client/commonwell-api.ts

```diff
@@ -93,6 +93,7 @@ export interface CommonWellAPI {
   getNetworkLinks(meta: RequestMetadata, patientId: string): Promise<PatientNetworkLinkResp>;
   deletePatient(meta: RequestMetadata, id: string): Promise<void>;
   queryDocuments(meta: RequestMetadata, patientId: string): Promise<DocumentQueryResponse>;
+  queryDocumentsFull(meta: RequestMetadata, patientId: string): Promise<DocumentQueryFullResponse>;
```

Comment:
[On this other PR ](https://github.com/metriport/metriport/pull/246#discussion_r1174427060)we talked about having only one "query documents" function, returning all data - if we're not addressing this now, maybe we can create a ticket so we don't forget about it?

Reply:
After investigating this, it wasn't worth the refactor in places in the code that need *only* the doc refs, so I kept it separate

Reply:
For API/SDK simplicity, IMO it looks better to have one function and filter the results than having diff ones and having to choose which one to use in certain situations

But NABD

Reply:
Yeah both seem valid - right now though we have the flexibility to not need to do filtering in places where we don't care about the operation outcome (quite a few)